core/common/lib/json-ld-lib/src/main/java/org/eclipse/edc/jsonld/util/JacksonJsonLd.java

```diff
@@ -36,6 +36,7 @@ public void setupModule(SetupContext context) {
             }
         };
         mapper.registerModule(module);
+        mapper.enable(DeserializationFeature.ACCEPT_SINGLE_VALUE_AS_ARRAY);
```

Comment:
should this be part of a separate PR for the sake of clarity? It could also deserve a test

Reply:
This was already present in the code base, I just move it here centrally. 
A test make sense :+1: